backend/meeting/package.json

```diff
@@ -19,12 +19,13 @@
         "cors": "^2.8.5",
         "dotenv": "^16.3.1",
         "express": "^4.18.2",
-        "jsonwebtoken": "^9.0.2",
-        "mongoose": "^7.5.3"
+        "jsonwebtoken": "^9.0.2"
     },
     "devDependencies": {
         "cross-env": "^7.0.3",
         "jest": "^29.7.0",
+        "mongodb-memory-server": "^9.1.6",
+        "mongoose": "^8.2.1",
```

Comment:
good question thought it should be both, maybe both mongodb-memory-server and mongoose should be normal.

Reply:
No, only the mongoose package. MongoDB-memory-server is only used to for development (testing) of the application to create a mongoDB behind the scenes, am I right? We dont use this in production. But since we use our mongoose schemas throughout the whole application, even in production. It should probably be in the regular dependencies list.